creme/stream/test_sql.py

```diff
@@ -1,5 +1,4 @@
-import sql
-from sql import iter_sql 
+from . import sql
 from sqlalchemy import create_engine, MetaData 
```

Comment:
Okay now you need to put your test code inside a function. I invite you check out other test files to get an idea of how to set this up. Feel free to ping me if you need help.